shell/platform/darwin/ios/BUILD.gn

```diff
@@ -153,6 +143,142 @@ shared_library("create_flutter_framework_dylib") {
   ]
 }
 
+static_library("ocmock") {
+  configs -= [ "//build/config/compiler:chromium_code" ]
+  cflags = [
+    "-fvisibility=default",
+    "-F/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform//Developer/Library/Frameworks/",
+    "-mios-simulator-version-min=13.0",
+    "-Wno-misleading-indentation",
+  ]
+  sources = [
+    "../../../../../third_party/ocmock/Source/OCMock/NSInvocation+OCMAdditions.h",
```

Comment:
I don't know if it works, but instead of digging through all these source, can't the dependent just have a deps = [ "//third_party/ocmock" ] here? (searching through other .gn file examples in third_party)

Reply:
We can't because ocmock doesn't have a GN file that specifies how to build it.  We can't add one there either because that isn't a repo we control.  We can clone it at some point.

Reply:
There is one though https://chromium.googlesource.com/chromium/src/+/master/third_party/ocmock/BUILD.gn. It's probably our DEPS needing to get it from chromium rather than the upstream github directly.

Reply:
We can't use chromium's ocmock because it is a source_set not a static_library.  I can try a static_library based on that source_set but i've had problems with static_libraries in the past.  I'd prefer to do this as a separate PR if you don't mind.

Reply:
ah ok, feel free to do it in a separate PR. 

That source_set is default public visibility, I suppose we can still make a static_library here that just deps = [ "//third_party/ocmock" ] if we clone the right repo.